.github/workflows/ci.yml

```diff
@@ -54,7 +54,8 @@ jobs:
           glome.c glome.h glome_test.c
     - name: check glome-cli format
       run: |
-        clang-format --Werror --dry-run --style=google cli/main.c
+        clang-format --Werror --dry-run --style=google \
+          cli/main.c cli/commands.c cli/commands.h
```

Comment:
That was an explicit decision for dependencies, iirc. We should discuss whether it makes sense to keep it for formatting. The newly added `yapf` and `pylint` use wildcards, fwiw.

Reply:
It wasn't as much a decision there: meson requires this for all inputs to ensure it can write out an accurate build graph for ninja. For formatting I'd say file endings would be fine. I'd go as far as **/*.c and **/*.h all need to be formatted correctly. Obviously that breaks down once you have the first exception, though. ;-)

Reply:
Switched to using a wildcard for formatting.